controllers/ctltools.js

```diff
@@ -66,9 +69,9 @@ const sanitizeToolInputs = (body, create= false) => {
 };
 
 const getUpdateHandler = (req, res, next) => {
-    req.body = sanitizeToolInputs(req.body);
+    req.body = sanitizeToolInputs(req.params.id, req.body);
 
-    api(req, { version: 'v3' }).put(`/tools/${req.params.id}`, {
+    api(req, { version: 'v3' }).post(`/tools/${req.params.id}`, {
```

Comment:
The post it what you want for a update?

Reply:
yes because we want to increase version number in future, so it's not idempotent.

Reply:
ok